common/static/js/vendor/requirejs/text.js

```diff
@@ -0,0 +1,394 @@
+/**
```

Comment:
It is so great that you've got RequireJS Text working. We should consider how to start using it everywhere, especially when we enable RequireJS Optimizer.